src/index.html

```diff
@@ -9,6 +9,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a data-qa="hover"
+              href="#"
+              class="nav__link"
+            >Laptops & computers</a>
```

Comment:
```suggestion
            <a 
              data-qa="hover"
              href="#"
              class="nav__link"
            >
              Laptops & computers
            </a>
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/ca7d794e-ac4d-48c0-8265-0b60ee03bc6f)